font_api/registry.lua

```diff
@@ -60,6 +60,7 @@ local function get_default_font()
 
 		-- If failed, chose first font
 		if default_font == nil then
+			-- luacheck: ignore 512
 			for _, font in pairs(font_api.registered_fonts) do
 				default_font = font
 				break
```

Comment:
This loop should be replaced with:
```lua
default_font = next(font_api.registered_fonts)
```
Also the comment is wrong, this will choose whichever font happens to get picked first.